apps/summerfi-api/lib/sdk-router-function/src/context.ts

```diff
@@ -5,11 +5,12 @@ import { DeploymentIndex } from '@summerfi/deployment-utils'
 import { Deployments } from '@summerfi/core-contracts'
 import { OrderPlannerService } from '@summerfi/order-planner-service/implementation'
 import { SwapService } from '@summerfi/swap-service'
+import type { GetQuote } from '@summerfi/simulator-service'
 
 export type ContextOptions = CreateAWSLambdaContextOptions<APIGatewayProxyEventV2>
 
 export type Context = {
-  provider: undefined | string
+  getQuote: undefined | GetQuote
   deployments: DeploymentIndex
   orderPlannerService: OrderPlannerService
   swapService: SwapService
```

Comment:
I think I could refactor refinance strategy to use SwapService at it also includes getQuote. What do you think?

Reply:
@paszkowskiDamian it might be cleaner yes please, as getQuote here is basically a duplication

Reply:
In my other PR (https://github.com/OasisDEX/summerfi-monorepo/pull/101) that has already changed:
- There is a `swaps` entry in the router with `getSwapQuoteExactInput` and `getSwapDataExactInput` handlers
- Also Swap service is in the router context, that can be passed as a dependency to simulator, and it already includes the proper handler